README.md

```diff
@@ -82,15 +82,15 @@ var format = d3.format(".2s");
 
 ## API Reference
 
-<a name="format" href="#format">#</a> d3.<b>format</b>(<i>specifier</i>)
+<a name="format" href="#format">#</a> d3.<b>format</b>(<i>specifier</i>) [<>](https://github.com/d3/d3-format/blob/master/src/formatDefault.js "Source")
 
 An alias for [*locale*.format](#locale_format) on the [default locale](#formatDefaultLocale).
 
-<a name="formatPrefix" href="#formatPrefix">#</a> d3.<b>formatPrefix</b>(<i>specifier</i>, <i>value</i>)
+<a name="formatPrefix" href="#formatPrefix">#</a> d3.<b>formatPrefix</b>(<i>specifier</i>, <i>value</i>) [<>](https://github.com/d3/d3-format/blob/master/src/formatPrefixAuto.js "Source")
 
 An alias for [*locale*.formatPrefix](#locale_formatPrefix) on the [default locale](#formatDefaultLocale).
 
-<a name="locale_format" href="#locale_format">#</a> <i>locale</i>.<b>format</b>(<i>specifier</i>)
+<a name="locale_format" href="#locale_format">#</a> <i>locale</i>.<b>format</b>(<i>specifier</i>) [<>](https://github.com/d3/d3-format/blob/master/src/locale.js#L50 "Source")
```

Comment:
The correct link is https://github.com/d3/d3-format/blob/master/src/locale.js#L18